src/RobotFrames.hpp

```diff
@@ -8,7 +8,11 @@
 #include "base/samples/Joints.hpp"
 #include "base/samples/RigidBodyState.hpp"
 #include "kdl/tree.hpp"
-#include "robot_framesTypes.hpp"
+#include "robot_frames/robot_framesTypes.hpp"
+
+#include <kdl/chainfksolvervel_recursive.hpp>
```

Comment:
not sure. Is  `chainfksolvervel_recursive.hpp` really needed?

Reply:
removed